tests/integration/test_broken_detached_part_clean_up/test.py

```diff
@@ -1,360 +0,0 @@
-
-    node.query(f"DETACH TABLE {table}")
-    node.query(f"ATTACH TABLE {table}")
-
-    node.wait_for_log_line(
-        "Removing detached part deleting_all_123_456_7",
-        timeout=90,
-        look_behind_lines=1000000,
-    )
-    node.wait_for_log_line(
-        f"Removed broken detached part {expect_broken_prefix}_all_3_3_0 due to a timeout",
-        timeout=10,
-        look_behind_lines=1000000,
-    )
-    node.wait_for_log_line(
-        "Removed broken detached part unexpected_all_42_1337_5 due to a timeout",
-        timeout=10,
-        look_behind_lines=1000000,
-    )
-
-    result = node.exec_in_container(["ls", path_to_detached])
-    print(result)
-    assert f"{expect_broken_prefix}_all_3_3_0" not in result
-    assert "all_1_1_0" in result
-    assert "trash" in result
-    assert "broken_all_fake" in result
-    assert "covered-by-broken_all_12_34_5" in result
-    assert "unexpected_all_42_1337_5" not in result
-    assert "deleting_all_123_456_7" not in result
-
-    node.query(
-        f"ALTER TABLE {table} DROP DETACHED PART 'covered-by-broken_all_12_34_5'",
-        settings={"allow_drop_detached": 1},
-    )
-    result = node.exec_in_container(["ls", path_to_detached])
-    assert "covered-by-broken_all_12_34_5" not in result
-
-    node.query(f"DROP TABLE {table} SYNC")
-
-
-def test_remove_broken_detached_part_merge_tree(started_cluster):
-    node1.query(
-        """
-        CREATE TABLE
-            mt(id UInt32, value Int32)
-        ENGINE = MergeTree() ORDER BY id
-        SETTINGS
-            merge_tree_enable_clear_old_broken_detached=1,
-            merge_tree_clear_old_broken_detached_parts_ttl_timeout_seconds=5;
-        """
-    )
-
-    for i in range(4):
-        node1.query(
-            f"INSERT INTO mt SELECT number, number * number FROM numbers ({i * 100000}, 100000)"
-        )
-
-    remove_broken_detached_part_impl("mt", node1, "broken-on-start")
-
-
-def test_remove_broken_detached_part_replicated_merge_tree(started_cluster):
-    node1.query(
-        f"""
-        CREATE TABLE
-            replicated_mt(date Date, id UInt32, value Int32)
-        ENGINE = ReplicatedMergeTree('/clickhouse/tables/replicated_mt', '{node1.name}') ORDER BY id
-        SETTINGS
-            merge_tree_enable_clear_old_broken_detached=1,
-            merge_tree_clear_old_broken_detached_parts_ttl_timeout_seconds=5,
-            cleanup_delay_period=1,
-            cleanup_delay_period_random_add=0,
-            cleanup_thread_preferred_points_per_iteration=0;
-        """
-    )
-
-    for i in range(4):
-        node1.query(
-            f"INSERT INTO replicated_mt SELECT toDate('2019-10-01'), number, number * number FROM numbers ({i * 100000}, 100000)"
-        )
-
-    remove_broken_detached_part_impl("replicated_mt", node1, "broken")
-
-
-def test_store_cleanup(started_cluster):
-    node1.query("CREATE DATABASE db UUID '10000000-1000-4000-8000-000000000001'")
-    node1.query(
```

Comment:
@alexey-milovidov, this test was testing a different feature, please add it back

Reply:
https://github.com/ClickHouse/ClickHouse/pull/55605